util.h

```diff
@@ -176,22 +176,30 @@ typedef struct {
    selectable. These spare bits allow for additional features for the varargs
    stuff or ABI compat test flags in the future.
 */
-#define HSm_APIVERLEN       0x0000001F                  /* perl version string won't
+#define HSm_APIVERLEN       0x0000001F                  /* perl version
+                                                           string won't
                                     be more than 31 chars */
 #define HS_APIVERLEN_MAX    HSm_APIVERLEN
-#define HSm_XSVERLEN        0x0000FF00                  /* if 0, not present, dont check, die if over 255*/
+#define HSm_XSVERLEN        0x0000FF00                  /* if 0, not present,
+                                                   dont check, die if over 255*/
 #define HS_XSVERLEN_MAX           0xFF
 /* uses var file to set default filename for newXS_deffile to use for CvFILE */
 #define HSf_SETXSUBFN       0x00000020
-#define HSf_POPMARK         0x00000040                  /* popmark mode or you must supply ax and items */
-#define HSf_IMP_CXT         0x00000080                  /* ABI, threaded/MULTIPLICITY, pTHX_ present */
-#define HSm_INTRPSIZE       0xFFFF0000                  /* ABI, interp struct size */
-/* A mask of bits in the key which must always match between a XS mod and interp.
+#define HSf_POPMARK         0x00000040                  /* popmark mode or you
+                                                     must supply ax and items */
+#define HSf_IMP_CXT         0x00000080                  /* ABI,
+                                         threaded/MULTIPLICITY, pTHX_ present */
+#define HSm_INTRPSIZE       0xFFFF0000                  /* ABI, interp struct
+                                                           size */
```

Comment:
Ah. This one is actually a "difficult case" of a few issues piling up together that I am already working on. The first unforunate  decisions is that the line up code is lining up values like '0x0000040' with defines for code expressions. This is then forcing the columns wider than they should be, which is then eating up the space for where the comment goes, which is then leaving the comment wrapping logic only 20 columns to work with, and this comment contains "threaded/MULTIPLICITY", which is a very long word, so then it cant line it up with the rest without breaking the line limit, and then it gives up.

FWIW, this is not the last step, and the last step is intended to try to help with this case. However, I am trying to approach this a different way, and come up with a better partitioning algorithm/heuristic so lists of numbers like this (which very often have sidebar comments) are lined up independently and thus put the name and value tighter together so that there is more space to wrap the side-bar comment. Having said that there comes a certain point where I might address this by just removing the "/" in "threaded/MULTIPLICITLY" so this can be wrapped more easily.